features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-add/template.jag

```diff
@@ -2,10 +2,10 @@
     var APIUtil = org.wso2.carbon.apimgt.impl.utils.APIUtil;
     var apiPath = "";
     var goBack = "";
-    if(session.get('apiPath') != null){
+    if (session.get('apiPath') != null) {

```

Comment:
We can change this as in [1] as well :)
[1] - https://github.com/wso2/carbon-apimgt/pull/5392#discussion_r271594297

Reply:
Resolved